app/board.rb

```diff
@@ -0,0 +1,93 @@
+class board
+    attr_accessor :squares
+
+    def initialize(player_1, player_2)
+        @player_1 = player_1
+        @player_2 = player_2
+        @squares = Array.new(64, " ")
+        start_pos
+
+
+        @positions = [  [a,1], [a,2],[a,3],[a,3],[a,4],[a,5],[a,6],[a,7], [a,8], 
+                        [b,1], [b,2],[b,3],[b,3],[b,4],[b,5],[b,6],[b,7], [b,8],
+                        [c,1], [c,2],[c,3],[c,3],[c,4],[c,5],[c,6],[c,7], [c,8], 
+                        [d,1], [d,2],[d,3],[d,3],[d,4],[d,5],[d,6],[d,7], [d,8], 
+                        [e,1], [e,2],[e,3],[e,3],[e,4],[e,5],[e,6],[e,7], [e,8], 
+                        [f,1], [f,2],[f,3],[f,3],[f,4],[f,5],[f,6],[f,7], [f,8], 
+                        [g,1], [g,2],[g,3],[g,3],[g,4],[g,5],[g,6],[g,7], [g,8], 
+                        [h,1], [h,2],[h,3],[h,3],[h,4],[h,5],[h,6],[h,7], [h,8]]
+        
+    end
+
+    def start_pos
+        #####################Player_1 Pieces############################
+
+        @squares[48] = @player_1.pawn.piece
+        @player_1.pawn1.position = [g,1]
+        @squares[49] = @player_1.pawn.piece
+        @player_1.pawn2.position = [g,2]
+        @squares[50] = @player_1.pawn.piece
+        @player_1.pawn3.position = [g,3]
+        @squares[51] = @player_1.pawn.piece
+        @player_1.pawn4.position = [g,4]
+        @squares[52] = @player_1.pawn.piece
+        @player_1.pawn5.position = [g,5]
+        @squares[53] = @player_1.pawn.piece
+        @player_1.pawn6.position = [g,6]
+        @squares[54] = @player_1.pawn.piece
+        @player_1.pawn7.position = [g,7]
+        @squares[55] = @player_1.pawn.piece
+
+        #####################Player_2 Pieces############################
+
+        @player_1.pawn8.position = [g,8]
```

Comment:
Looks like you're still referring to `player_1` here. 

Looks like a lot of very similar, repeated code here. Thats usually a hint that it's not as DRY as it could be. How would this look converted to two loops, one for each player?